mapiter_go1.11.go

```diff
@@ -0,0 +1,42 @@
+// +build !go1.12
```

Comment:
License header (please also in race_test.go, we forgot it there).

Reply:
done